cli/.js/src/main/scala/aqua/run/RunCommand.scala

```diff
@@ -110,16 +103,33 @@ object RunCommand extends Logging {
       .flatMap(_.exports.map(e => Chain.fromSeq(e.funcs.values.toList)).getOrElse(Chain.empty))
       .find(_.funcName == funcName)
 
+  // Creates getter services for variables. Return an error if there is no variable in services
+  // and type of this variable couldn't be optional
+  def createGetters(vars: List[(String, Type)], services: Map[String, ArgGetterService]): ValidatedNec[String, List[ArgGetterService]] = {
+    vars.map {
+      (n, argType) =>
+        val serviceOp = services.get(n)
+        (serviceOp, argType) match {
+          case (None, _) => Validated.invalidNec(s"Unexcepted. There is no service for '$n' argument")
```

Comment:
Users don't know that there should be _services_ for function arguments... This doesn't tell them anything useful. 

But that error seems impossible overall.

You can eliminate this error if you store `Type` in `ArgGetterService`. And I think you might eliminate this whole function as well.